### PR TITLE
Toast library and example toast

### PR DESCRIPTION
**Pull request by srkirkland**
Thought i would get this out early, and if we like it I can start adding it into all components.  This library is pretty cool because we can pass a promise and it'll handle the spinner and updating, in addition to normal notifications.

### CODE REVIEW

**Review by sprucely — APPROVED** [review]
I like it. Do you think it would be helpful to wrap in a hook that returns something like `{ isLoading: bool, isSuccess: bool, error: any }`?

**Comment by srkirkland** [comment]
> I like it. Do you think it would be helpful to wrap in a hook that returns something like `{ isLoading: bool, isSuccess: bool, error: any }`?

Interesting idea, not sure how to actually build it.  I'll keep it in mind to come back to after getting this lib integrated.

**Review by jSylvestre — APPROVED** [review]
I like the look more than what we have for Peaks

File: Harvest.Web/ClientApp/src/Quotes/QuoteContainer.tsx [hunk 1]
```diff
@@ -150,10 +151,17 @@ export const QuoteContainer = () => {
       body: JSON.stringify(quote),
     });
 
+    toast.promise(request, {
+      loading: "Saving Quote",
+      success: "Quote saved.",
+      error: "Something went wrong, please try again",
```
Comment (jSylvestre) [inline Harvest.Web/ClientApp/src/Quotes/QuoteContainer.tsx:157]: @srkirkland I tried to throw an exception and return a bad request. Neither displayed the error.

**Comment by srkirkland** [comment]
closes #383

**Review by sprucely — APPROVED** [review]
Looks good, and BadRequests are giving toast errors.

**Review by jSylvestre — APPROVED** [review]
If you want to test the error, go to a ticket details page an reply with no text.

**Pull request merged**